gradle/scripts/testSetup.gradle

```diff
@@ -35,6 +35,7 @@ subprojects {
       }
       testLogging {
         events "started","skipped","passed","failed"
+        exceptionFormat "full"
```

Comment:
There isn't as far as I know, i needed to add this to gain some insight on failed tests since I don't think there's a way to access the gradle test html after tests.